srecord/input/filter/nibble_swap.h

```diff
@@ -0,0 +1,80 @@
+//
+// srecord filter that exchanges upper and lower nibble of each byte, e.g. "0xA6" => "0x6A"
```

Comment:
Please replace with a copyright assertion to a legal name (rather than username/handle). See another similar files for examples. 

`// srecord - Manipulate EPROM load files`
`// Copyright (C) 2023 Firstname Lastname`